resources/schemas/dbscripts/postgresql/cds-21.003-21.004.sql

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TABLE cds.import_virus_metadata_all
+(
+    cds_virus_id INTEGER NOT NULL,
+    virus VARCHAR(250) NOT NULL,
+    virus_full_name VARCHAR(250),
+    virus_backbone VARCHAR(250),
+    virus_host_cell VARCHAR(250),
+    virus_plot_label VARCHAR(250),
+    virus_type VARCHAR(250),
+    virus_species VARCHAR(250),
+    clade VARCHAR(250),
+    neutralization_tier VARCHAR(250),
+    container ENTITYID NOT NULL,
+
+    CONSTRAINT PK_ImportVirusMetadataAll PRIMARY KEY (cds_virus_id),
+    CONSTRAINT UQ_ImportVirusMetadataAll UNIQUE (virus_full_name)
```

Comment:
Should the unique constraint include container also?

Reply:
yup